coreax/coreset.py

```diff
@@ -58,6 +59,14 @@ class Coreset(eqx.Module, Generic[_Data]):
     nodes: Data = eqx.field(converter=as_data)
```

Comment:
I forgot the reasoning behind having the coreset be referred to as nodes rather than just coreset

Reply:
You can also think of the `nodes` are the information to parametrize the coreset/coresubset, and the `coreset` property is the way to materialize the real coreset data from this parametrization. In the case of `Coresubset` the nodes are the (weighted) indices, and the `coreset` property materializes the data points referenced by these indices (giving them the same weight as the index).

Reply:
Ah I see, makes sense!